_targets.R

```diff
@@ -118,6 +119,9 @@ NADP_sb_id <- '57e2ac2fe4b0908250045981'
 ## FORESCE list of FORESCE years
 FORESCE_years <- c('1940', '1950', '1960', '1970', '1980', '1990', '2000')
 
+## Path to synoptic sampling sites
+syn_file <- 'C:/Users/jsmith/OneDrive - DOI/Shared Documents - GS-WMA-IWP-PUMP/40.2 Data-driven salinity/Data/DRB-Inland/DRB_2021_synoptic_sites_by_transect_20210819.xlsx'
```

Comment:
I don't have access to this file. Rather than including your OneDrive address here, I'd suggest we instead add that file to the "Data" folder on Sharepoint and then include a comment in the code that the user has to manually download that file and place it in "1_fetch/in". I view that option as preferable to committing this file, especially since there probably won't be much active development on this branch.

Reply:
Sorry about that link. It is in our Sharepoint Data folder. Do you see it [here](https://doimspp.sharepoint.com/sites/usgs-GS-WMA-IWP-PUMP/Shared%20Documents/Forms/AllItems.aspx?CT=1631626506661&FolderCTID=0x01200053D2763891272743A0A7D8B9E952F717&id=%2Fsites%2Fusgs%2DGS%2DWMA%2DIWP%2DPUMP%2FShared%20Documents%2F40%2E2%20Data%2Ddriven%20salinity%2FData%2FDRB%2DInland&viewid=23f2c2aa%2Dd9c1%2D4fd9%2Daf8c%2D23ed3cf885a0)

Reply:
That's a good plan to switch to using `1_fetch/in` for this file

Reply:
I added this file to 1_fetch/in, added the file name to .gitignore, and added a check that the file exists